MyCryptoHQ/MyCrypto
# Simplify custom node URL

Closes #1123.

### Description

This changes custom nodes to go from being a url + port to just being a URL. Most people who are dealing with custom nodes know to add the port, and right now the current implementation just socks the port on the end, which doesn't work with path'd nodes. This may be somewhat more error prone, but should result in a more flexible user experience.

Side note: The change in fields in the modal requires a follow up PR to deal with modal sizing. That should be up momentarily.

### Changes

* Removes port field from custom nodes
* Reworks the way custom nodes are displayed in dropdown
    * Before it would show chainid + URL. Now it shows custom network name + custom node name.

### Steps to Test

1. Run the app on `custom-node-url` branch
2. Go to add a custom node
3. Specify your desired URL. If you don't know one, just take one from the static node configs file.
4. Confirm it works.

## Screenshots

### Old Dropdown Display vs New

<img width="342" alt="screen shot 2018-02-19 at 12 45 47 pm" src="https://user-images.githubusercontent.com/649992/36391518-d201590a-1574-11e8-9469-85b1f16ce306.png">

<img width="225" alt="screen shot 2018-02-19 at 12 48 36 pm" src="https://user-images.githubusercontent.com/649992/36391519-d52df688-1574-11e8-8273-535961029076.png">
